pkgs/development/web/flyctl/default.nix

```
@@ -32,9 +32,13 @@ buildGoModule rec {
   '';
 
   postCheck = ''
-    go test ./... -ldflags="-X 'github.com/superfly/flyctl/internal/buildinfo.buildDate=1970-01-01T00:00:00+0000'"
+    go test ./... -ldflags="-X 'github.com/superfly/flyctl/internal/buildinfo.buildDate=1970-01-01T00:00:00Z'"
   '';
 
+  passthru.tests.version = testers.testVersion {
+    command = "flyctl version";
```

```suggestion
    package = flyctl;
    command = "flyctl version";
```

it needs the package

```
[nix-shell:~/.cache/nixpkgs-review/pr-172428]$ nix-build nixpkgs/ -A flyctl.tests
error: 'testVersion' at /home/davidak/.cache/nixpkgs-review/pr-172428/nixpkgs/pkgs/build-support/testers/default.nix:7:5 called without required argument 'package'

       at /home/davidak/.cache/nixpkgs-review/pr-172428/nixpkgs/pkgs/development/web/flyctl/default.nix:38:28:

           37|
           38|   passthru.tests.version = testers.testVersion {
             |                            ^
           39|     command = "flyctl version";
```